tests/driver_at/app.config.test

```diff
@@ -0,0 +1,20 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+# dependencies of AT driver
+CONFIG_MODULE_ISRPIPE=y
+CONFIG_MODULE_ISRPIPE_READ_TIMEOUT=y
```

Comment:
Why are these specified here? Arent they part of the driver dependencies?

Reply:
Yes, that's why they have to be enabled to be able to use the driver

Reply:
Sorry dumb question, a bit too used to make.